docs/usage/configuration-options.md

```diff
@@ -2229,7 +2229,15 @@ In most cases it would not be recommended, but there are some cases such as Dock
 
 ## vulnerabilityAlerts
 
-Use this object to customise PRs that are raised when vulnerability alerts are detected (GitHub-only).
+Renovate can read from GitHub's Vulnerability Alerts and customize Pull Requests accordingly.
+For this to work, you must first ensure you have enabled "Dependency graph" and "Dependabot alerts" under the "Security & analysis" section of the repository's "Settings" tab.
```

Comment:
GitHub docs has some pages that we might want to link to:

- https://docs.github.com/en/code-security/supply-chain-security/about-the-dependency-graph#enabling-the-dependency-graph
- https://docs.github.com/en/github/administering-a-repository/managing-security-and-analysis-settings-for-your-repository (Has advice on public and private repos, so we can choose which one works with Renovate bot)

Reply:
We can link to each of these from "Dependency graph" and "Dependabot alerts" respectively

Reply:
```suggestion
For this to work, you must first ensure you have enabled "[Dependency graph](https://docs.github.com/en/code-security/supply-chain-security/about-the-dependency-graph#enabling-the-dependency-graph)" and "[Dependabot alerts](https://docs.github.com/en/github/administering-a-repository/managing-security-and-analysis-settings-for-your-repository)" under the "Security & analysis" section of the repository's "Settings" tab.
```

Reply:
@rarkins Is the suggestion above what you wanted?